### DISCUSSION TITLE
Migration to release verison of Avalonia 11

### DISCUSSION THREAD

**Discussion by mixonomicon** [AvaloniaCommunity/Material.Avalonia#263] [General]
Hi all,

We are in the process of planning our conversion of WPF LOBs into AvaloniaUI. We decided to focus on Avalonai 11, now that is fully  released. One thing about our WPF apps is that majority of them use Material.Wpf. hence, we imagine that migrating to Material.Avalonia would play out the best. I see there was a lot of work done on 3.0_major_update branch, relating to avalonia 11, so I imagine you are already close to it. 

So, a few points:
- What and when is the orientational plan to release the final 3.0 version for Avalonia 11?
- Until such time can we/should we use nightly builds to be ready?
- Is the prerelease nuget package usable at all? We added it to the project, it bound to 11, but when we ran it, some things didnt work (like datepicker)

**Comment by SKProCH** [comment]
Hello, @the-black-wolf 

Currently we support 11 Avalonia, but not all features are implemented yet. Currently you definitely need to use pre-releases, because it just has the most recent code with the latest fixes, etc. It's called pre-releases because we are still fixing and improving our library, but I don't think, we will introduce any major breaking changes, etc. So, you should definitely use the pre-releases for Avalonia 11 for now. 

As for the 3.0 release, I can't say something, but we will release asap as we support all default Avalonia controls. But I (and other maintainers) don't have a lot of time to develop for now, but I am trying to resolve issues and implement everything we need. If you are missing something, you can raise issues, and I'll try to prioritize it. 

DatePicker is the first thing I plan to fix.

**Reply by SKProCH** [reply]
Also, you can add a standard Avalonia theme (default or fluent) and use it as the fallback for missing controls. These controls will definitely not look materialistic, but I think it's better than nothing at the moment.

**Reply by cesarchefinho** [reply]
I think releasing a final version would be ideal even without all supported controls because some applications prevent the use of pre-released packages.

 And as more controls are supported, release version 3.1, 3.2 and etc...

**Reply by SKProCH** [reply]
@cesarchefinho if you interested - 3.0.0 released just now.
